src/NexusMods.App.UI/WorkspaceSystem/Context/DownloadsContext.cs

```diff
@@ -0,0 +1,3 @@
+namespace NexusMods.App.UI.WorkspaceSystem;
+
+public record DownloadsContext : IWorkspaceContext;
```

Comment:
Downloads will need an optional Loadout context in the future, do you want to add it now?
Maybe we need to discuss it a bit with Design?

Reply:
> Downloads will need an optional Loadout context in the future

What do you mean?

Reply:
> When opening the DL page in a Loadout Workspace it should automatically be filtered for that workspace.

That'd be for the page context `InProgressPageContext`. This is `DownloadsContext` which is a `WorkspaceContext`.

Reply:
Either way, it's not needed right now.